readme.md

```diff
@@ -1,14 +1,95 @@
 # Inexor
 
-Inexor is a fork of the quake-like FPS Cube 2: Sauerbraten.
-It adds a lot of functionality and tries to stay open for improvements and suggestions.
-The task is to be more flexible and create an environment where developement is easy and fast and creativity can prosper.
+Inexor is a fork of the open-source First-Person-Shooter Cube 2: Sauerbraten, a fast-paced shooting game featuring an ingame map editor.  
+In contrast to Sauerbraten, Inexor adds a lot of functionality and strives to stay open to improvements and suggestions.  
+The goal of this project is to be more flexible and create an environment where development is easy, fast and where creativity can prosper.
 
-You will find more information in our wiki.
+## How is Inexor organized?
 
+We are a non-hierarchical organisation. This means we are simply a group of people with different ideas working together without a leader making all the decisions. Any one of us is free to work on the particular things he/she wants to.
```

Comment:
slash gendering is terrible.

English has the singular they, which is awesome!